embedding/esp8266/readme.md

```diff
@@ -0,0 +1,68 @@
+### About
+
+Files in this folder (embedding/esp8266) are copied from
+examples/project_template of esp_iot_rtos_sdk and modified for JerryScript.
+You can view online from [this](https://github.com/espressif/esp_iot_rtos_sdk/tree/master/examples/project_template) page.
+
+
+### How to build JerryScript for ESP8266
+
+#### 1. SDK
+
+Follow [this](https://github.com/seanshpark/esp8266-docs/wiki/Preparation-and-setting-build-environment)
```

Comment:
Great guide! What do this about moving it into root source directory of the project? i.e `./docs/ESP-PREREQUISITES.md`